elm/Swole/Types/Movement.elm

```diff
@@ -0,0 +1,5 @@
+module Swole.Types.Movement exposing
+    ( Movement
+    )
+
+type alias Movement = String
```

Comment:
I'll "newtype" this in Elm, instead of the type alias, to be a bit more typesafe:

```elm
type Movement
    = Movement String
```

If you do this, be sure to update the export to include the data constructor.

Reply:
I'm fully expecting this type to become significantly more complex in the future. This current typealias is just a stop-gap for now.

Reply:
That sounds good, especially if the expectation is that it'll change in the (near) future